airbyte-integrations/connectors/source-teradata/acceptance-test-config.yml

```diff
@@ -0,0 +1,8 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-teradata:dev
+acceptance_tests:
+  spec:
+    tests:
+      - spec_path: "src/test-integration/resources/expected_spec.json"
+        config_path: "src/test-integration/resources/dummy_config.json"
```

Comment:
For `config_path` you need to provide a link to a file that contains credentials that will be used to run the integration tests. These are by convention put in a folder name `secrets` which are git ignored so that they are not accidentally checked in. Refer [Acceptance Tests Reference](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference/) and Postgres Source [implementation](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-postgres/acceptance-test-config.yml).

Reply:
sure thing!